apps/src/templates/sectionProgress/VirtualizedSummaryView.jsx

```diff
@@ -57,30 +79,49 @@ export default class VirtualizedDetailView extends Component {
     // Subtract 1 to account for the student name column.
     const stageIdIndex = columnIndex-1;
 
+    // Override default cell style from multigrid
+    let cellStyle = {
+      ...style,
+      ...styles.cell,
+    };
+    // Alternate background colour of each row
+    if (studentStartIndex%2 === 1) {
+      cellStyle = {
+        ...cellStyle,
+        backgroundColor: color.background_gray,
+      };
+    }
+
     return (
-      <div className={styles.Cell} key={key} style={style}>
```

Comment:
I removed className because I couldn't figure out what it was used for, but let me know if I need to add it back in.

Reply:
I think it's used here: https://github.com/bvaughn/react-virtualized/blob/master/source/MultiGrid/MultiGrid.example.css#L1 for the MultiGrid Cell styling, but you've used all new styles so I agree it's fine to remove.

Reply:
Ohhh I actually think flexbox is important so I'll keep it.